Multiple fixes to thermal jet nozzles

Cholerix opened the pull request:
Fix for multiple bugs and inconsistencies with thermal jet nozzles.

- The radius of the Wyvern thermal ramjet nozzle is 1.25 m. However, the size was set to 2.5 m in the `ThermalNozzleController` module. The weight was also too high for a 1.25 m part.
    - Radius set to 1.25 m everywhere. `exitArea` in `ThermalNozzleController` lowered accordingly.
    - Weight set to 0.75 t. I chose this value since the physical size of the model falls between the short thermal turbojet (0.5 t) and the thermal turbo ramjet (1.0 t).
- Removed duplicate atmosphere+fuel propellant types that had excessive performance multipliers. These were only used by the Phoenix afaik.
- Unified propellant types across all thermal jet nozzles. This means the Phoenix can now run on intake air, I presume this is how it was intended. All nozzles can now also use hydrogen based fuels.
- Removed duplicate liquid carbon monoxide fuel that had the same properties as liquid neon.
- Updated the descriptions of all three thermal jet nozzles to indicate their performance difference. It wasn't until I looked at the code that I understood them.

I noticed upgrades don't work. This is because `upgradeTechReq` in `ThermalNozzleController` is set to true, when it should be set to a technology. I did not change this, as I can't tell if this is intentional. The upgrade still appears in "Experimental Rocketry" but does nothing as far as I can tell.

sswelm reviewed (changes requested):
They are not mend for LFO but mend to be used with Thermal Turbojet VTOL, using AirIntake.  I would prefer you do not change this.

Cholerix commented:
> They are not mend for LFO but mend to be used with Thermal Turbojet VTOL, using AirIntake. I would prefer you do not change this.

I assume you mean the fuel types? The VTOL can still use the intake+fuel mixes as it can use LFO fuels. It was never able to use the fuels I removed:
![Thermal Turboramjet fuels](https://user-images.githubusercontent.com/33205074/148207918-a31bc7c5-3cde-4c87-9f28-86418640b1c2.png)
Notice the different names and the ISP multipliers. I removed the fuel of the phoenix in this picture. The VTOL was not changed.
